package.json

```diff
@@ -1,74 +1,49 @@
-{
-    "name": "pylint",
-    "displayName": "Pylint",
-    "description": "%extension.description%",
-    "version": "2022.5.0-dev",
-    "preview": true,
-    "serverInfo": {
-        "name": "Pylint",
-        "module": "pylint"
-    },
-    "publisher": "ms-python",
-    "license": "MIT",
-    "homepage": "https://github.com/Microsoft/vscode-pylint",
-    "repository": {
-        "type": "git",
-        "url": "https://github.com/microsoft/vscode-pylint.git"
-    },
+{  
```

Comment:
The sorting looks good, but we may need to revert the first changes you made that moved the "name", "displayname", "description", etc. fields from the beginning of the file. 

If you run `git log`, you can find the list of your commit IDs. If you get your first commit ID, you can try doing `git revert <first_commit_Id>`, and that would allow you to revert those first changes!